plugin/pki/venafi_util.go

```diff
@@ -112,25 +112,44 @@ var venafiTestTPPConfigAllAllow = map[string]interface{}{
 	"tpp_password":      os.Getenv("TPP_PASSWORD"),
 	"zone":              os.Getenv("TPP_ZONE"),
 	"trust_bundle_file": os.Getenv("TRUST_BUNDLE"),
+	"auto_refresh":      true,
 }
 
-var venafiTestTPPConfigRestricted = map[string]interface{}{
+var venafiTestTPPConfigNoRefresh = map[string]interface{}{
```

Comment:
Are you sure?

Reply:
What the problem here? Oh, it's just git, you can see venafiTestTPPConfigRestricted bellow